maat-court-data-api/src/main/java/gov/uk/courtdata/eform/service/EformStagingDAO.java

```diff
@@ -0,0 +1,20 @@
+package gov.uk.courtdata.eform.service;
+
+import gov.uk.courtdata.eform.dto.EformStagingDTO;
+
+/**
+ * The responsibility of this class is to provide an abstraction on top of the Database layer.
+ * Specifically to allow CRUD operations on the HUB.EFORMS_STAGING table.
+ */
+public interface EformStagingDAO {
```

Comment:
Can we change the interface to IEformService?

Reply:
Changed to EFormStagingService for clarity and not added the 'I' as its a dated method case from C++ users that migrated over to Java.